webpack/ForemanInventoryUpload/Components/IpsObfuscationSwitcher/__tests__/IpsObfuscationSwitcherActions.test.js

```diff
@@ -0,0 +1,21 @@
+import { testActionSnapshotWithFixtures } from '@theforeman/test';
+import API from 'foremanReact/API';
+import { handleToggle } from '../IpsObfuscationSwitcherActions';
+import { handleToggleResponse } from '../IpsObfuscationSwitcher.fixtures';
+
+jest.mock('foremanReact/API');
+API.post.mockImplementation(async () => handleToggleResponse);
+
+const fixtures = {
+  'should handleToggle': () => handleToggle(),
+  'should handleError': () => {
+    API.post.mockImplementationOnce(() =>
+      Promise.reject(new Error('test error'))
+    );
+
+    return handleToggle();
```

Comment:
can you pass the `currentIpsObfuscationEnabled` param here and on also on line 10?
although there are mocks it's good to pass it, instead of `undefined`